src/Entity/Annotations.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+
+class Persistent extends Annotation {}  
+  
+/** @Target("class") */
+class Table extends Annotation {}  
+
+/** @Target("property") */
+class Column extends Annotation {
+    public $name;
+    public $type;
+}
+
+/** @Target("property") */
+class Id extends Annotation {}
+
+class Join extends Annotation {
```

Comment:
Sim Jack, tbm não gosto muito de fazer isso. Essas são anotações customizadas que criei para gerar o script to banco de dados. Como são só anotações  e não classes de verdade, resolvi deixar tudo em um único lugar. Se julgar mais conveniente posso extrair para arquivos indepentendes